codemods/jscodeshift/next/13/remove-get-static-props/index.ts

```diff
@@ -33,86 +33,256 @@ function findLastIndex<T>(
 	return -1;
 }
 
-export const findGetStaticPropsFunctions: ModFunction<File, 'read'> = (
+/**
+ * factories
+ */
+
+const generateStaticParamsMethodFactory = (j: JSCodeshift) => {
+	return j.exportNamedDeclaration(
+		j.functionDeclaration.from({
+			async: true,
+			body: j.blockStatement([j.returnStatement(j.arrayExpression([]))]),
+			id: j.identifier('generateStaticParams'),
+			comments: [j.commentLine(' TODO: implement this function')],
+			params: [],
+		}),
+	);
+};
+
+const serverHookParamsFactory = (j: JSCodeshift) => {
+	return j.identifier.from({
+		name: 'params',
+		typeAnnotation: j.tsTypeAnnotation(
+			j.tsTypeReference(j.identifier('PageParams')),
```

Comment:
it this type imported?

Reply:
Okay, i forgot to add type alias, it should be
```
//should be defined by user
type PageParams = {}
type PageProps = {
 params: PageParams
 }
```